handler_success.go

```diff
@@ -193,6 +193,8 @@ func (s *SuccessHandler) RecordHit(r *http.Request, timing int64, code int, resp
 			// mw_redis_cache instead? is there a reason not
 			// to include that in the analytics?
 			if responseCopy != nil {
+				responseCopy.Body = respBodyReader(r, responseCopy)
```

Comment:
You also need to set ContentLength header to 0 here as well

Reply:
It will be set in `respBodyReader` function so it is not necessary to set again.

Reply:
ok